Module 8/Ex 3/Sources/Interpreter.cpp

```diff
@@ -0,0 +1,62 @@
+#include "Interpreter.hpp"
+#include "IncrementPtr.hpp"
+#include "DecrementPtr.hpp"
+#include "IncrementByte.hpp"
+#include "DecrementByte.hpp"
+#include "PrintByte.hpp"
+#include "SetByte.hpp"
+#include "JumpForward.hpp"
+#include "JumpBackward.hpp"
+#include <algorithm>
+#include <iostream>
+
+Interpreter::Interpreter() : arr{}, p(arr.data())

```

Comment:
not really safe to go directly to underlying array.
Is an iterator not possible ?
Or a smart pointer maybe ?